nap/utils.py

```diff
@@ -48,10 +34,15 @@ def make_url(base_url, params=None, add_slash=None):
 
         # If we're given an non-string iterable as a params value,
```

Comment:
The placement of these comments is surprising to me.  I would expect them to be above `flatten_params`, not `safe_encode`.